commands/build.go

```diff
@@ -0,0 +1,48 @@
+package commands
+
+import (
+	log "github.com/sirupsen/logrus"
+	build "github.com/tarantool/cartridge-cli/build_project"
+	"github.com/tarantool/cartridge-cli/project"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(buildCmd)
```

Comment:
Is it described in some guidelines?

Alternative variant is export command description and add it directly in main

Reply:
In the cobra README: https://github.com/spf13/cobra#create-additional-commands

Reply:
Thanks for explanation